padl/transforms.py

```diff
@@ -1723,6 +1761,15 @@ def load(path):
     return transform
 
 
+def _zip_load(path: Union[Path, str]):
+    """Load a transform from a compressed '.padl' file. """
+    # we can't use a context because the files need to exist when using / saving again
```

Comment:
What do you mean by this comment exactly? It looks like you are using a `with` context below?

Reply:
When using the context, the temporary directory is deleted when leaving. This doesn't work unfortunately as the files are needed for code inspection / saving. Thus we can't use the context.

Reply:
Could you add a note, specifying that the context we can't use refers to the `TemporaryDirectory`?